libs/asset-viewer/src/lib/components/asset-search-refine/asset-search-refine.component.ts

```diff
@@ -57,6 +62,15 @@ export class AssetSearchRefineComponent implements OnInit, OnDestroy, AfterViewI
   readonly assetKindFilters$ = this.store.select(selectAssetKindFilters);
   readonly workgroupFilters$ = this.store.select(selectWorkgroupFilters);
 
+  readonly activeFilters$ = combineLatest([
+    this.usageCodeFilters$,
+    this.geometryCodeFilters$,
+    this.manCatLabelFilters$,
+    this.languageFilters$,
+    this.assetKindFilters$,
+    this.workgroupFilters$,
+  ]).pipe(map((a) => a.flat(1).filter((filter) => filter.isActive)));
```

Comment:
This reduces the iterations significantly due to filtering first and flattening second.
Doesn't change behavior, but the order bothered me somewhat.
```suggestion
  ]).pipe(map((fss) => fss.flatMap((fs) => fs.filter((f) => f.isActive))));
```